content/5.open-source/2.self-hosting.md

```diff
@@ -62,48 +63,42 @@ Here are four different use cases. Please refer to their APP_HOMEPAGE_URL settin
 Run on your own computer
 
 ```yaml [docker-compose.yml]
-version: '3'
+networks:
+  keydb:
+  mongo:
 
 services:
-heyform:
-  image: heyform/community-edition:latest
-  restart: always
-  volumes:
-    # Perist uploaded images
-    - ./assets:/app/static/upload
-  depends_on:
-    - mongo
-    - redis
-  ports:
```

Comment:
Heyform requires an external network port

Reply:
I'm confused about what you highlighted vs what you commented but if you like I'll update it regardless bare with me.

Reply:
didn't mean to overcorrect and delete ALL of the port binds my mistake.